crates/router/tests/connectors/paypal.rs

```diff
@@ -567,7 +565,7 @@ async fn should_fail_payment_for_incorrect_cvc() {
         .unwrap();
     assert_eq!(
         response.response.unwrap_err().message,
-        r#"[{"field":"/payment_source/card/security_code","value":"12345","location":"body","issue":"INVALID_PARAMETER_SYNTAX","description":"The value of a field does not conform to the expected format."}]"#.to_string(),
+        "description - The value of a field does not conform to the expected format., value - 12345 ; ",
```

Comment:
Error message doesn't mention that it is related to card_security_code(CVV). previously it was